scripts/verifyPatternflyVersions.js

```diff
@@ -47,7 +47,7 @@ async function verifyPatternflyVersions() {
   if (process.argv[2] === '--fix') {
     mismatchedVersions.forEach(([dep, versions]) => {
       const highestVersion = Object.keys(versions)
-        .sort(semver.compare)
+        .sort((a, b) => semver.compare(a.replace(/^\^/, ''), b.replace(/^\^/, '')))
```

Comment:
Is this change unrelated?

Reply:
I added it so I could run `node scripts/verifyPatternflyVersions.js --fix` to update react-console's out-of-date `@patternfly/react-core`.

Reply:
Super thanks